models/unit/unit.go

```diff
@@ -170,11 +172,12 @@ func (u *Type) CanBeDefault() bool {
 
 // Unit is a section of one repository
 type Unit struct {
-	Type    Type
-	NameKey string
-	URI     string
-	DescKey string
-	Idx     int
+	Type     Type
+	NameKey  string
+	URI      string
+	DescKey  string
+	Idx      int
+	MaxPerms perm.AccessMode
```

Comment:
So MaxPerms = max(Authorize,Authorize) which are AccessModes? 

Why not just AccessMode?